config/environments/development.rb

```diff
@@ -41,7 +41,7 @@
   config.assets.quiet = true
 
   # Raises error for missing translations
-  # config.action_view.raise_on_missing_translations = true
+  config.action_view.raise_on_missing_translations = true
```

Comment:
Another minor configuration change here so (in development, anyway) that the app will 💣 out when there's a missing translation. Currently, the app will render markup and throw a notice in the HTML which… isn't… terribly… noticeable.